css/properties/display.json

```diff
@@ -737,22 +743,22 @@
                 }
               ],
               "firefox_android": {
-                "version_added": null
+                "version_added": "57"
               },
               "ie": {
                 "version_added": false
               },
               "opera": {
-                "version_added": false
+                "version_added": "52"
               },
               "opera_android": {
-                "version_added": null
+                "version_added": "52"
               },
               "safari": {
-                "version_added": false
+                "version_added": "11.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "11.1"
```

Comment:
Why 11.1 and not 11.3?

Reply:
11.3 is the version of iOS that contains the 11.1 version of Safari (as far as I can tell).

Reply:
Makes sense. Thanks.